streampark-common/src/main/java/org/apache/streampark/common/enums/FlinkSqlValidationFailedType.java

```diff
@@ -20,14 +20,19 @@
 import java.io.Serializable;
 
 public enum FlinkSqlValidationFailedType implements Serializable {
+
   /** Basic test failed (such as null, etc.) */
   VERIFY_FAILED(1),
+
   /** syntax error */
   SYNTAX_ERROR(2),
+
   /** unsupported dialect */
   UNSUPPORTED_DIALECT(3),
+
   /** unsupported sql command */
   UNSUPPORTED_SQL(4),
+
   /** Not at the end of ";" */
   ENDS_WITH(5);
 
```

Comment:
`private final int value` -> `private final int failedType`;

Reply:
get it